api/src/pcapi/core/offerers/api.py

```diff
@@ -1498,8 +1498,14 @@ def _get_query(offer_class: type[offers_api.AnyOffer]) -> BaseQuery:
         return sa.select(sa.func.jsonb_object_agg(sa.text("status"), sa.text("number"))).select_from(
             sa.select(
                 sa.case(
-                    (offer_class.isActive.is_(True), "active"),  # type: ignore [attr-defined]
-                    (offer_class.isActive.is_(False), "inactive"),  # type: ignore [attr-defined]
+                    (sa.and_(offer_class.isActive.is_(True), offer_class.is_expired.is_(False)), "active"),  # type: ignore [attr-defined]
```

Comment:
Je te vois confus sur les `type: ignore`, tu peux ruser si tu ne veux pas les augments :angel: 
```suggestion
                    (sa.and_(offer_class.isActive, sa.not_(offer_class.is_expired)), "active"),
```

Reply:
j'ai une autre erreur avec ce changement : 
```
 error: Value of type variable "_CLE" of function cannot be "object"  [type-var]
```

Reply:
ce n'est pas grave, laisse comme tu avais mis, avec les type: ignore